src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.spec.ts

```diff
@@ -53,13 +58,16 @@ describe('ConstraintsPanelComponent', () => {
         // Estimated cost in $ per acre
         estimatedCost: ['', [Validators.min(0), Validators.required]],
         // Max cost of treatment for entire planning area
-        maxCost: ['', Validators.min(0)],
+        maxCost: [{ value: '', disabled: true }, Validators.min(0)],
       }),
       physicalConstraintForm: fb.group({
```

Comment:
Looks like we are defining a FormGroup on the test with validators and fields, which is great, but I don't see the same definition on the actual component, just `@Input() constraintsForm: FormGroup | undefined;`

Should we have the same Form Builder definition on the component? Im not sure if I understand why the form is marked as an input, seems like part of the form definition is coming from the parent and part is being extended here, which can be quite tricky to debug and follow.

What do you think? Im still learning about the codebase so Im probably missing context :)

Reply:
Discussed with Pablo:
- This structure was inherited with the code — with the changes we've made probably not necessary anymore
- Agreed it would be cleaner to have the form builder on the component and then output the user input back to the parent for scenario creation
- Will add a TODO for a future pr for now